src/config/index.ts

```diff
@@ -201,7 +203,9 @@ export const BO_ID = `/:${ROUTE_PARAM_BENEFICIAL_OWNER_ID}`;
 export const TRUST_ID = `/:${ROUTE_PARAM_TRUST_ID}`;
 export const TRUSTEE_ID = `/:${ROUTE_PARAM_TRUSTEE_ID}`;
 export const GET_COMPANY_OFFICERS_PAGE_SIZE = 100;
-
+export const REVIEW_BENEFICIAL_OWNER_INDEX_PARAM = "?index=";
+export const REVIEW_BENEFICIAL_OWNER_REVIEW_PARAM = "&review=true";
```

Comment:
So far we have not used the review query arg. I though it would be needed for navigation but so far that not the case. Should we remove it?

Reply:
Yes review removed